src/CMakeLists.txt

```diff
@@ -27,7 +27,9 @@ set(
         view/opengl_interfacing/vertex.cpp
         controller/entity_translator.cpp
         controller/terrain_translator.cpp
-        model/height_map.cpp ../include/model/height_map.hpp)
```

Comment:
This is the correct place to put files for compilation, I have moved your files here while resolving merge conflicts, please be aware that you have to uncheck the default place where clion tries to put source files for compilation while adding a new class. You need to add the path here manually, and only to the cpp file, hpp files don't need to be compiled.